scripts/bundle.config.js

```diff
@@ -72,6 +72,9 @@ const config = {
         options: {
           presets: [
             ['@babel/preset-env', {forceAllTransforms: true}]
+          ],
+          plugins: [
```

Comment:
Since this is a JS file, nothing prevents adding comments here about the setup...

E.g, Are we also including the `@babel/runtime` module in the bundle to ensure the runtime references that are left as `imports` by `@babel/transform-runtime` are resolved? 

And what happens if the app includes multiple bundles including `@babel/runtime`?